### ISSUE TITLE
Should warmer cooler alerts interrupt other alerts?

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#241]
Now that we are hearing these alerts from bug fixes in https://github.com/phetsims/friction/issues/234, I wonder if we need to tweak how they are interrupting others.

@terracoda, I felt like it sounded best to hear the warmer alerts interrupt other alerts (like grabbed alerts), but not for the cooler alerts. Let me know what you think!

**Comment by terracoda** [comment]
This is working pretty good. 
I think the warming and cooling responses should interrupt other responses like the grab and release responses.

In addition, I find that after the sim is super hot the cooling responses start quire late after interaction stops. Often the the thermometer is halfway down before any cooling responses start.

I think there a few super-fast-super-hot responses that are piling up causing the delay.

**Comment by zepumph** [comment]
Over in #240, we will move the "reset the sim to make more observations" This will help shorten the breakaway alert, and may get us more cooling alerts earlier. After that we will rereview this one.

**Comment by zepumph** [comment]
I'm actually about having made "warming" alerts interrupt, because not I don't think we are getting any break away alerts. Let's talk about that more here.

**Comment by zepumph** [comment]
https://github.com/phetsims/friction/issues/240 has been done, and the reset hint is no longer on the "super fast" alert.

I think we need to think about the warmer alerts now.

**Comment by terracoda** [comment]
@terracoda requested that we try cooling responses on end drag.
That seems to be working. Will re-listen before closing.

**Comment by terracoda** [comment]
We will test after we fix break away response, which getting interrupted by warming responses.

**Comment by zepumph** [comment]
One commit coming from changes that we worked on today during our meeting this morning. This is generally to try to get cooling alerts occurring a bit faster. This will also be further improved by a change over in https://github.com/phetsims/friction/issues/242 where the break away alert is going to clear the queue before alerting (removing any superfluous warming alerts).

**Comment by terracoda** [comment]
@Matthew-Moore240, I am unassigning myself from this issue. I think you will be able to review this issue once the related issues are committed. Feel free to re-assign if you need anything from me.

**Comment by zepumph** [comment]
https://github.com/phetsims/friction/issues/242 is mostly done. Please review this issue too now.

**Comment by Matthew-Moore240** [comment]
I am hearing the breakaways a lot more often which is good. I still am rarely getting a cooling response but I am not sure if thats another issue so I guess ill pass this back to @zepumph to close or continue the discussion if I am missing something.

**Comment by zepumph** [comment]
> I still am rarely getting a cooling response but I am not sure if thats another issue

Yes, I think that is mostly because of #253, but a byproduct of this prioritized break-away alert is that it will always be complete, and no cooler alerts will come until after it finishes. Closing
